### DISCUSSION TITLE
Support for SolidJS

### DISCUSSION THREAD

**Discussion by reasonablekeith** [konvajs/konva#1842] [General]
There is an npm library [solid-konva](https://www.npmjs.com/package/solid-konva), which provides a similar declarative component experience for Solid as [react-konva](https://www.npmjs.com/package/react-konva) does for React.

However, the solid version has almost no downloads and hasn't received an update for 2 years now. I am currently building a tool using Solid and Konva, but because of this seeming lack of support, we've written our own component library for the subset of Konva APIs we're using.

I'll reach out to the owner of solid-konva too, but wanted to ask this community...

- Why, when the Konva website headlines component support for React, Svelte & Vue, is the no endorsement for a Solid library?
- Would the community make use of a well maintained component library for Solid, and would the Konva team be interested in endorsing/supporting it?

We actually started our build with React/Konva, but we ended up migrating to Solid when we were hitting performance issues with React. It seems to me like Solid/Konva go great together, and I'm surprised there isn't more 'official support'.

I'd be happy to support publishing a library if the interest was there.

**Comment by lavrton** [comment]
Hello.

> Why, when the Konva website headlines component support for React, Svelte & Vue, is the no endorsement for a Solid library?

It is just a lot less popular. So there is no much demand.

> Would the community make use of a well maintained component library for Solid, and would the Konva team be interested in endorsing/supporting it?

I am interested in having a well-maintained version of `solid-konva` library. I am happy to include it into official docs and mentions and make some demos with it.

Personally, I would not be able to support it much from code/upgrades perspective, as I don't use solid in daily work. But I am ready to help with publishing, code review, docs.
